### DISCUSSION TITLE
Current connection still working? Syncing different tabs/pages

### DISCUSSION THREAD

**Discussion by soccerarenadev** [wevm/wagmi#789] [Q&A]
Is there a way to ask if the current connection is still up?

I'm trying to sync different tabs. When another tab disconnect, I can't find a way to ask wagmi if the current connection is still working. However, useConnect returns me outdated information.

Can I force to check is the connection is still up?

**Comment by tmm** [comment]
`useConnect` is a mutation hook. You likely want to use [`useAccount`](https://wagmi.sh/docs/hooks/useAccount) to see if there is still a connection.
